tap_github/schemas/team_members.json

```diff
@@ -22,6 +22,24 @@
           "string"
         ]
```

Comment:
Can we use Owner.json's ref for team_members.json?

Reply:
This schema contained _sdc_repository, hence didn't use it. However, added the _sdc_repository in the shared schema as well.

Reply:
Do we need _sdc_repository at other places?

Reply:
No we dont